modules/ROOT/pages/questions/question-04.adoc

```diff
@@ -0,0 +1,13 @@
+4. The `toc` plugin has been documented as _new and improved_ because
+   of the following:
+
+   *the `toc` plugin has modernised its output. It now uses the ES2020
+   library and targets ES2018. As well, it now uses
+   https://terser.org[terser] to improve performance.*
+
+   It felt like at least some explanation for moving the plugin to
```

Comment:
@lostkeys

Reply:
Table of contents and imagetools both have no user-facing changes in this release. We simply lifted the open source plugin and made it premium.